packages/ui/src/proposals/components/ProposalDetails/ProposalDetails.tsx

```diff
@@ -72,6 +74,24 @@ export const ProposalDetails = ({ proposalDetails }: Props) => {
   const detailsRenderStructure = useMemo(() => getDetailsRenderStructure(proposalDetails), [proposalDetails])
 
   const additionalDetails = useMemo(() => {
+    if (gracePeriod) {
+      return [
+        {
+          renderType: 'NumberOfBlocks',
+          label: 'Gracing Period',
+          value: gracePeriod,
+        },
+      ] as RenderNode[]
+    }
+    if (exactExecutionBlock) {
+      return [
+        {
+          renderType: 'NumberOfBlocks',
+          label: 'Exact Execution Block',
+          value: exactExecutionBlock,
+        },
+      ] as RenderNode[]
+    }
```

Comment:
Here `exactExecutionBlock` is not actually a number of blocks but a block height so `NumberOfBlocks` is not the right component. Instead it should show something like the [BlockTime](https://pioneer-2-storybook-joystream.vercel.app/?path=/story/common-blocktime--block-time-view) component.